gini/gini-capture-sdk-android
# Feature/use camera x

Using CameraX for controlling the camera. This is a big one and I still need to update the tests.

The `CameraInterface` was updated (along with all relevant code) to make it camera api agnostic. This allows us to switch between the old camera api implementation and the new CameraX one.

Also using CameraX to check the camera requirements. Here the old `CameraHolder` was renamed `OldCameraApiHolder` and `CameraHolder` was transformed into a camera api agnostic interface. This too allows us to switch between the camera apis used to check the camera requirements.

### How to test
Install the example apps and take pictures of documents and check the extractions.

### Ticket 
https://ginis.atlassian.net/browse/PIA-1481
